<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Filters</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="MsgLossWarning">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QwtTextLabel" name="FrameInformationLable">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionDevice"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionClose"/>
    <addaction name="actionDatabase"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionGraphicWindow"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuNew"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDevice">
   <property name="text">
    <string>Device</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionGraphicWindow">
   <property name="text">
    <string>GraphicWindow</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
